### ISSUE TITLE
PixelPlanet timelapse maker

### ISSUE THREAD

**Issue by msenya25** [Topinambur223606/PixelPlanetTools#5]
### Usage
`timelapse.exe X1 Y1 X2 Y2 CanvasType FileExportName FileFormat T1 T2 DateInFrame [FPS]`

- **X1 Y1 X2 Y2** - top left and bottom right coordinates
- **Canvas type** "E" - earth map "M" - moon map
- **FileExportName** - name of the export file
- **FileFormat** - file export format, only .gif and .mp4 formats
-  **T1 T2** - start and end of timelapse
-  **DateInFrame** "D" - one day in the frame "E" - every saved frame in timelapse
- **FPS** - Frames per second. Non-required, 60 FPS by default
Example:
`timelapse.exe -512 -512 511 511 M timelapse .gif 08.01.2020 08.02.2020 E 60`

**Comment by Topinambur223606** [comment]
Record visualizer at the moment allows to get minute deltas. Theoretically, it's possible to make video or gif from delta pictures, but it's not that simple if you want to get daily based frames.

Also I didn't work with moon canvas but I think it's quite similar to regular.
To implement it, watcher should be also possible to record changes at moon map; fortunately, it does not require logging in for observing.
To sum up - thank you for the idea.

**Comment by msenya25** [comment]
Well, I thought take images from historical view, and make from that images a video or gif. But unfortunately, in historical view you can't download a image.

**Comment by Topinambur223606** [comment]
I totally forgot about historical view; I think it's possible to use it, I'll think about that possibility.

**Comment by 4ChanMeta** [comment]
There's already a timelapse maker, on the official pixelplanet github. Go there and find the utils tab, and there's a function called historyDownload.py

you can make your timelapse with that
